persistence/cosmosdb/index_outboxstorageformat_CosmosDB_[3.2,).partial.md

```diff
@@ -0,0 +1,9 @@
+A default [synthetic partition key](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/synthetic-partition-keys) will be used for all incoming messages, in the format `{endpointName}-{messageId}`, if not explicitly [overwritten](/persistence/cosmosdb/transactions.md#specifying-the-partitionkey-to-use-for-the-transaction) at runtime.
+
+> [!NOTE]
+> The [default partition key must be overwritten](/persistence/cosmosdb/transactions.md#specifying-the-partitionkey-to-use-for-the-transaction) whenever the message handler creates or updates business documents in CosmosDB. To guarantee atomicity, explicitly set the Outbox partition key to match the partition key of your business document (e.g., an `OrderId` or `CustomerId`). This is a requirement for including both the business document and the Outbox record in the same [Cosmos DB transactional batch](https://learn.microsoft.com/en-us/azure/cosmos-db/partitioning-overview). Conversely, for simplicity, you can use the default partition key when a handler's logic does not involve persisting business data.
```

Comment:
I wouldn't say that it "must" be overwritten - perhaps "should"? Also - giving the example of a key such as "OrderId" would cause the outbox issue that we have just solved by changing the default key to be synthetic. 
So perhaps best not to give an example? Unless we can come up with one that doesn't cause a problem.